docs/source/gbd2019_models/causes/maternal_disorders/index.rst

```diff
@@ -292,8 +292,17 @@ Simulants who experience an incident case of maternal disorders and occupy the p
 
 .. math::
 
-  YLDs per non-fatal maternal disorders case = \frac{\text{ylds}_{c366} - \text{ylds}_\text{s182,s183,s184}}{\text{incidence_rate}_{c366} - (ACMR - csmr_\text{c366} + csmr_\text{c366} / incidence_\text{c366})}
+  \text{YLDs per non-fatal maternal disorders case} = 
 
+  \frac{\text{ylds}_{c366} - \text{ylds}_\text{s182,s183,s184}}{\text{incidence_rate}_{c366} - (ACMR - csmr_\text{c366} + csmr_\text{c366} / incidence_\text{c366})}
+
+.. warning::
+
+  The above equation yielded a negative value in initial attempts. We've implementated the following placeholder (which may result in an underestimation of maternal disorder YLDs):
+
+  .. math::
+
+    \text{YLDs per non-fatal maternal disorders case} = \frac{\text{ylds}_{c366} - \text{ylds}_\text{s182,s183,s184}}{\text{incidence_rate}_{c366}}
```

Comment:
We are currently calculating the YLDs as `(maternal_ylds - anemia_ylds) / maternal_incidence`, but this doesn't have that division. I assume the code ought to update to this?

Reply:
No, those are consistent! It's just the horrible latex syntax doesn't make that obvious. Here is the rendered version (noting to myself that I should always include a screenshot like this for ease of review) 
![image](https://user-images.githubusercontent.com/54918472/158912249-0594434d-a066-4dd7-b2ac-d852b0f12857.png)

Reply:
Oh perfect. I indeed misparsed the LaTeX!